onesignalclient/notification.py

```diff
@@ -87,35 +97,49 @@ def contents(self):
 
     @contents.setter
     def contents(self, value):
+        self._contents = json.dumps(self._validate_content_dict(value))
+    
```

Comment:
Can you remove these unwanted whitespaces?

Ref. 'W293:blank line contains whitespace'